infrastructure/cli/auth/provider.go

```diff
@@ -47,7 +47,15 @@ func (a *CliAuthenticationProvider) Authenticate(ctx context.Context) (string, e
 	err := a.authenticate(ctx)
 	if err != nil {
 		log.Err(err).Str("method", "Authenticate").Msg("error while authenticating")
-		a.errorReporter.CaptureError(err)
+
+		// No need to report the error when it's due to the user not authenticating
+		const timedOutExitCode = 2
+		exitErr, ok := err.(*exec.ExitError)
+		if !ok || exitErr.ExitCode() != timedOutExitCode {
```

Comment:
Is there a chance CLI will produce exit code 2, when it's other problem than the timeout?